src/pages/workspaces/workspace/analysis/Analyses.ts

```diff
@@ -540,8 +539,10 @@ export const BaseAnalyses: FC<AnalysesProps> = ({
           appDataDisks,
           refreshAnalyses,
           analyses,
+          // @ts-expect-error
```

Comment:
I'm not sure, but this may have exposed some bugs. Could someone in IA check this out?

Reply:
Ah yeah, I actually fix this in an open PR

Reply:
You can either update `AnalysisModal`  to take the prop types here, or I can after you merge this (which you'll probably prefer to avoid updating related tests)

Reply:
Thanks. Yeah, I just wanted to flag this in case it was an actual bug. I'll go ahead and merge this.